commands/xrevrange.md

```diff
@@ -14,54 +14,6 @@ enough to send:
 
     XREVRANGE somestream + - COUNT 1
 
-## Iterating with XREVRANGE
```

Comment:
do you think maybe there's a reason to keep it for the benefit of redis 5.0 and 6.0 users?

Reply:
Added back as a subsection in XRANGE